network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/settings/geoswitching/GeoSwitchingContentProvider.kt

```diff
@@ -16,52 +16,80 @@
 
 package com.duckduckgo.networkprotection.impl.settings.geoswitching
 
-import com.duckduckgo.di.scopes.ActivityScope
+import com.duckduckgo.app.global.DispatcherProvider
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.networkprotection.impl.configuration.WgVpnControllerService
 import com.duckduckgo.networkprotection.impl.settings.geoswitching.GeoSwitchingContentProvider.AvailableCountry
+import com.duckduckgo.networkprotection.store.NetPGeoswitchingRepository
+import com.duckduckgo.networkprotection.store.db.NetPGeoswitchingLocation
 import com.squareup.anvil.annotations.ContributesBinding
 import javax.inject.Inject
+import kotlinx.coroutines.withContext
 
 interface GeoSwitchingContentProvider {
```

Comment:
About naming, this is not really about geo switching but rather about getting the available VPN servers.
Should we rename to `VpnEgressServersProvider` or something like that?

Reply:
Renamed to `NetpEgressServersProvider`